Index: coquillo-1.9/libs.pri
===================================================================
--- coquillo-1.9.orig/libs.pri
+++ coquillo-1.9/libs.pri
@@ -11,6 +11,6 @@ UI_DIR = $$ROOT/build
 
 VERSION = 1.0
 
-target.path = $$PREFIX/lib/coquillo
+target.path = $$PREFIX/lib$${libsuff}/coquillo
 INSTALLS += target
 
Index: coquillo-1.9/src/src.pro
===================================================================
--- coquillo-1.9.orig/src/src.pro
+++ coquillo-1.9/src/src.pro
@@ -19,7 +19,7 @@ unix {
 	LIBS += -ltag
 
 	# Imports the lib path into the binary
-	QMAKE_RPATHDIR += $$PREFIX/lib/coquillo
+	QMAKE_RPATHDIR += $$PREFIX/lib$${libsuff}/coquillo
 
 	target.path = $$PREFIX/bin
 
